weather-app/README.md

```diff
@@ -0,0 +1,38 @@
+# Node.js HTTP Function
+
+Welcome to your new Node.js function project! The boilerplate function code can be found in [`index.js`](./index.js). This function will respond to incoming HTTP GET request. This example function is written asynchronously, returning a JSON value. It will take city name provided as query parameter from the url and return the weather data for the city.
+
+## Local execution
+
+Before executing the function follow these steps:
+1. Go to https://openweathermap.org/api
+2. Sign up and create an API_KEY.
+3. Copy and paste your API_KEY in env.local file under env folder.
+4. npm install
+5. npm start
+
+The parameter provided to the function endpoint at invocation is a `Context` object containing HTTP request information.
+
+```js
+function handleRequest(context) {
```

Comment:
what is the purpose of mentioning this here? you are actually doing something else in the body of the function?

Reply:
Removed from README.